cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -61,14 +61,14 @@ spec:
               then
                 exit 1;
               fi;
-              LAST_MODIFIED_DATE=`stat /var/log/fluentd-buffers | grep Modify | sed -r "s/Modify: (.*)/\1/"`;
-              LAST_MODIFIED_TIMESTAMP=`date -d "$LAST_MODIFIED_DATE" +%s`;
-              if [ `date +%s` -gt `expr $LAST_MODIFIED_TIMESTAMP + $STUCK_THRESHOLD_SECONDS` ];
+              touch -d "@$(date -d"${STUCK_THRESHOLD_SECONDS} seconds ago" '+%s')" /tmp/marker-stuck;
+              if [[ -z $(find /var/log/fluentd-buffers -type f -newer /tmp/marker-stuck -print -quit) ]];
```

Comment:
You probably want quotes around the command substitution...

Reply:
Done.